packages/lesswrong/components/votes/lwReactions/ReactionQuotesHoverInfo.tsx

```diff
@@ -62,25 +61,18 @@ const styles = (theme: ThemeType): JssStyles => ({
   }
 });
 
-const ReactionQuotesHoverInfo = ({react, voteProps, commentBodyRef, classes}:{
-  react: string, 
+const ReactionQuotesHoverInfo = ({react, quote, voteProps, commentBodyRef, classes}:{
+  react: EmojiReactName,
+  quote: QuoteLocator,
   voteProps: VotingProps<VoteableTypeClient>,
   commentBodyRef?: React.RefObject<ContentItemBody>|null,
   classes: ClassesType
 }) => {
-  const { ReactOrAntireactVote } = Components;
-  const extendedScore = voteProps.document?.extendedScore as NamesAttachedReactionsScore|undefined;
+  const { ReactOrAntireactVote, UsersWhoReacted } = Components;
+  const normalizedReactions = getNormalizedReactionsListFromVoteProps(voteProps);
 
-  const alreadyUsedReactions: NamesAttachedReactionsList = extendedScore?.reacts ?? {};
-  const usersWhoReacted = alreadyUsedReactions[react]
-
-  const allQuotesOrUndefined = uniq(usersWhoReacted?.flatMap(r => r.quotes))
-  const allQuotes = filter(allQuotesOrUndefined, q => q !== undefined) as string[]
-  
-  const quotesWithUsers = allQuotes.map(quote => {
-    const usersWhoReactedToQuote = usersWhoReacted?.filter(r => quote && r.quotes?.includes(quote))
-    return { quote, users: usersWhoReactedToQuote  }
-  })
+  const reactionsOfType: UserReactInfo[] = normalizedReactions?.reacts?.[react] ?? [];
+  const reactionsToQuote = reactionsOfType.filter(r => r.quotes?.[0] === quote);
```

Comment:
Should this be `r.quotes?.[0] === quote` or `r.quotes?.includes(quote)` (or something like that)?  It's possible I'm misunderstanding some part of the logic here

Reply:
Oh, nvm, I see what's coming out of `getNormalizedReactionsListFromVoteProps` will always only have a single quote in the array.  Might be better to refactor it so it's just not returning that in the form a single-item (or empty) array?